backend/src/main/java/com/cs471/studentLoanSystem/login/LoginSystem.java

```diff
@@ -0,0 +1,43 @@
+package com.cs471.studentLoanSystem.login;
+
+import java.net.URI;
+
+import com.cs471.studentLoanSystem.roles.StudentInfo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+@RestController
+public class LoginSystem {
+
+    @GetMapping("/login")
+    public String loginForm(Model model) {
+        model.addAttribute("LoginInfo", new LoginInformation());
+        return "LoginInfo";
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity<LoginResponse> loginForm(
+            @ModelAttribute LoginInformation information, Model model) {
+        model.addAttribute("LoginInfo", information);
+        LoginResponse response = new LoginResponse();
+        /* Fill out the response with data from the database */
+        response.setName("Andrew Gabler");
+        response.setAuthenticated(false);
+        response.setRole("Student");
+        StudentInfo info = new StudentInfo();
+        info.setAddress("1160 Dupont St");
+        info.setSchool("Kettering");
+        info.setStudentId(2);
+        response.setStudentInfo(info);
+        /*****************************************************/
+
+        URI uri =
```

Comment:
Why are we attempting to call another part of the controller?

You should be able to just set the return to `LoginResponse` instead of `ResponseEntity<LoginResponse>` so that you can remove this unnecessary step.